### ISSUE TITLE
To create level of abstraction for communication with database

### ISSUE THREAD

**Issue by aszulinski** [krro/arts#5]
Is it done yet? Or maybe is it still up to do?

**Comment by lukgad** [comment]
Nope, right now data access is pure mongo: https://github.com/krro/arts/blob/master/src/main/scala/com/kainos/ArtsRepository.scala

**Comment by krro** [comment]
Well, to be honest the interface of this class knows nothing about the mongo
